src/main/java/Flash.java

```diff
@@ -43,6 +35,44 @@ public static void unMarkTask(String input) {
         System.out.println("____________________________________________________________");
     }
 
+    public static void todo(String input){
+        String description = input.substring(5).trim();
+        Task task = new ToDo(description);
+        tasks.add(task);
+        System.out.println("____________________________________________________________");
+        System.out.println(" " + task);
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void deadline(String input) {
+        String[] parts = input.split(" /by ");
+        String description = parts[0].trim();
+        String by = parts[1].trim();
+        Task task = new Deadline(description, by);
+        tasks.add(task);
+        System.out.println("____________________________________________________________");
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  " + task);
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void event(String input) {
+        String[] parts = input.substring(6).split(" /from | /to");
```

Comment:
Avoid using magic numbers like substring(6).split. Used named constants for these numbers instead.